modules/layers/src/solid-polygon-layer/solid-polygon-layer.js

```diff
@@ -111,14 +116,20 @@ export default class SolidPolygonLayer extends Layer {
           positions64xyLow: {
             size: 2
           },
+          instancedPositions64xyLow: {
+            size: 2,
+            divisor: 1
```

Comment:
In luma.gl `divisor=1` is currently auto deduced if attribute name starts with `instance`. That particular inference is a bit subtle/magic though, so it makes perfect sense to explicitly state it here.